### PR TITLE
allow receive single HTTP measures using attribute values in plain, raw and json content-types

### PR DESCRIPTION
**Pull request by AlvaroVega**
issue https://github.com/telefonicaid/iotagent-json/issues/661

- [x] New endpoint for POST single measures: `/iot/json/attrs/:attrValue` allowing these content-types: `application/json`, `text/plain`, `application/octet-stream`
- [x] Doc
- [x] Test / examples

This PR enables send single measures like:

curl -i -X POST 'http://localhost:7897/iot/json/attrs/myattr1?i=disp4&k=6idn7z9cba3o57tgoascazdwd' -d '33' -H 'content-type: application/json'

curl -i -X POST 'http://localhost:7897/iot/json/attrs/myattr1?i=disp4&k=6idn7z9cba3o57tgoascazdwd' -d 'asdfasdfasd' -H 'content-type: text/plain'

curl -i -X POST 'http://localhost:7897/iot/json/attrs/myattr1?i=disp4&k=6idn7z9cba3o57tgoascazdwd' -d '01010011' -H 'content-type: application/octet-stream'

### CODE REVIEW

**Comment by AlvaroVega** [comment]
Default attribute type used by MQTT when a single measrue is receive from /json/apikey/id/attrs is `string`.
In this case, using HTTP, defaul type will be `None`

**Comment by fgalan** [comment]
Should a twin PR be done for IOTA-UL?

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1 +1,5 @@
+- Allow receive single HTTP attribute measures posted to `/iot/json/attrs/<attributeName>` with different content-types `application/json`, `text/plain` and `application/octet-stream`
+- Upgrade body-parser dep from 1.18.3 to 1.20.0
+- Upgrade express dep from 4.16.4 to 4.18.1
+- Do not use json parse strict for incoming http measures, commands and configuration
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:4]:
```suggestion
- Add: allow receive single HTTP attribute measures posted to `/iot/json/attrs/<attributeName>` with different content-types `application/json`, `text/plain` and `application/octet-stream`
- Upgrade body-parser dep from 1.18.3 to 1.20.0
- Upgrade express dep from ~4.16.4 to 4.18.1
- Fix: do not use json parse strict for incoming http measures, commands and configuration
```

**Comment by AlvaroVega** [comment]
> Should a twin PR be done for IOTA-UL?

This issue does not applies to iota-ul

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1 +1,5 @@
+- Add: allow receive single HTTP attribute measures posted to `/iot/json/attrs/<attributeName>` with different content-types `application/json`, `text/plain` and `application/octet-stream`
+- Upgrade body-parser dep from 1.18.3 to 1.20.0
+- Upgrade express dep from ~4.16.4 to 4.18.1
+- Fix: do not use json parse strict for incoming http measures, commands and configuration
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:4]: Issue #661 should be cited (not sure if in the "Add:" line, in the "Fix:" line or in both)

**Pull request merged**